sky/shell/gpu/direct/rasterizer_direct.h

```diff
@@ -24,19 +24,24 @@ namespace shell {
 class GaneshContext;
 class GaneshSurface;
 
-class RasterizerDirect : public Rasterizer {
+class RasterizerDirect : public Rasterizer, private rasterizer::Rasterizer {
```

Comment:
rasterizer::Rasterizer should be a public base class.  Also, Rasterizer should have the the base class.  We want this both for RasterizerMojo and for RasterizerDirect.